debian/apparmor-profile

```diff
@@ -97,6 +97,8 @@
     # telepathy-ofono needs to store a database for tracking pending messages
     owner @{HOME}/.local/share/telepathy-ofono/  rw,
     owner @{HOME}/.local/share/telepathy-ofono/** rwk,
+    # telepathy-ofono uses QTemporaryFile, which creates hardlinks, for its attachments
+    owner @{HOME}/.local/share/telepathy-ofono/attachments/** rwkl,
```

Comment:
Why not just add the `l` to the previous line?

Reply:
I want to give the least permissions possible